### PR TITLE
Update app name, urls, and copyright (extends #5)

### PR DESCRIPTION
**Pull request by andreif**
Extended version of #5

### CODE REVIEW

**Comment by andreif** [comment]
Should I change the `PRODUCT_NAME` to `GitX(N)` as well?

**Comment by andreif** [comment]
I have made a similar PR for the website https://github.com/gitx/gitx.github.io/pull/3
I haven't changed the name there, so it's still `GitX-dev`. Let me know if I should update it there as well.

File: Resources/English.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ﻿/* Localized versions of Info.plist keys */
 
-NSHumanReadableCopyright = "© Pieter de Bie, 2010";
\ No newline at end of file
+NSHumanReadableCopyright = "© GitX(N) developers, 2017";
```
Comment (jspiro) [inline Resources/English.lproj/InfoPlist.strings:3]: Per https://tldrlegal.com/license/gnu-general-public-license-v2 we can't do this. Can someone correct me?

File: Classes/Controllers/ApplicationController.m [hunk 2]
```diff
@@ -143,10 +143,10 @@ - (IBAction)showAboutPanel:(id)sender
 		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:gitversion, @"Version", nil]];
 
 	#ifdef DEBUG_BUILD
-		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:@"GitX-dev (DEBUG)", @"ApplicationName", nil]];
+		[dict addEntriesFromDictionary:[[NSDictionary alloc] initWithObjectsAndKeys:@"GitX(N) (DEBUG)", @"ApplicationName", nil]];
```
Comment (jspiro) [inline Classes/Controllers/ApplicationController.m:146]: Why not just go with GitX and no qualifier? Is there a system I'm unaware of?

GitX(NG) (next-gen) is another option?

Or... GitX(2021) or GitX(202X)? That'll make discovery way easier.

**Comment by hannesa2** [comment]
It would be nice, when you rebase to recent master to have a CI build. Thank you !

File: Resources/English.lproj/InfoPlist.strings (hunk 1 above)
Comment (hannesa2) [reply Resources/English.lproj/InfoPlist.strings:3]: My interpretation is (maybe I'm wrong) : we can do this 
image

our changes are **_available_** via git history

**Comment by hannesa2** [comment]
I rebased it https://github.com/gitx/gitx/pull/260 so I close this

**Pull request closed without merging**